src/main/java/net/fabricmc/loom/configuration/providers/minecraft/MinecraftVersionMeta.java

```diff
@@ -29,108 +29,35 @@
 import java.util.Map;
 import java.util.Objects;
 
-import com.google.gson.JsonObject;
-
 import net.fabricmc.loom.util.OperatingSystem;
 
 @SuppressWarnings("unused")
-public final class MinecraftVersionMeta {
-	private JsonObject arguments;
-	private AssetIndex assetIndex;
-	private String assets;
-	private int complianceLevel;
-	private Map<String, Download> downloads;
-	private String id;
-	private List<Library> libraries;
-	private JsonObject logging;
-	private String mainClass;
-	private int minimumLauncherVersion;
-	private String releaseTime;
-	private String time;
-	private String type;
-
-	public Download getDownload(String key) {
-		return getDownloads().get(key);
-	}
-
-	public JsonObject getArguments() {
-		return arguments;
-	}
-
-	public AssetIndex getAssetIndex() {
-		return assetIndex;
-	}
-
-	public String getAssets() {
-		return assets;
-	}
-
-	public int getComplianceLevel() {
-		return complianceLevel;
-	}
-
-	public Map<String, Download> getDownloads() {
-		return downloads;
-	}
-
-	public String getId() {
-		return id;
-	}
-
-	public List<Library> getLibraries() {
-		return libraries;
-	}
-
-	public JsonObject getLogging() {
-		return logging;
-	}
-
-	public String getMainClass() {
-		return mainClass;
-	}
-
-	public int getMinimumLauncherVersion() {
-		return minimumLauncherVersion;
-	}
-
-	public String getReleaseTime() {
-		return releaseTime;
-	}
-
-	public String getTime() {
-		return time;
-	}
-
-	public String getType() {
-		return type;
-	}
-
-	public final class AssetIndex extends Downloadable {
-		private String id;
-		private long totalSize;
-
-		public String getFabricId(String version) {
+public final record MinecraftVersionMeta(
```

Comment:
```suggestion
public record MinecraftVersionMeta(
```
They're always final.